BasicSample/ProjectSettings/ProjectSettings.asset

```diff
@@ -99,7 +99,7 @@ PlayerSettings:
   xboxEnableFitness: 0
   visibleInBackground: 1
   allowFullscreenSwitch: 1
-  fullscreenMode: 1
+  fullscreenMode: 3
```

Comment:
Should we make this change for RemotingSample as well?
I think this is a better default than fullscreen, since it's much easier to close

Reply:
Solved in latest iteration.  Changed all 3 projects to show in window instead of fullscreen.